docs/self-managed/deployment/helm/configure/authentication-and-authorization/microsoft-entra.md

```diff
@@ -251,7 +251,6 @@ global:
         audience: "<mgmt-identity-app-id>"
         initialClaimName: preferred_username
         initialClaimValue: "<the email address of your initial admin user>"
-        redirectUrl: "<IDENTITY_URL>"
```

Comment:
@jessesimpson36 Just want to confirm this looks correct to you -- if so I can approve and merge.

Reply:
This is correct. this option should be removed